velox/functions/lib/TimeUtils.h

```diff
@@ -104,4 +106,82 @@ struct InitSessionTimezone {
     timeZone_ = getTimeZoneFromConfig(config);
   }
 };
+
+/// Return day-of-year (DOY) of the first `dayOfWeek` in the year.
+///
+/// `dayOfWeek` is a 1-based weekday number starting with Sunday.
+///   (1 = Sunday, 2 = Monday, ..., 7 = Saturday).
+///
+/// If the `dayOfWeek` is Monday, it returns DOY of first Monday in
+/// the year. The returned DOY is a number from 1 to 7.
```

Comment:
> The returned DOY is a number from 1 to 7.

Does the meaning of the return value match `(1 = Sunday, 2 = Monday,..., 7 = Saturday)`?